indico/htdocs/sass/base/_palette.scss

```diff
@@ -22,6 +22,10 @@ $pastel-blue: #CDE1FF;
 $light-blue: #EDF3FD;
 $link: #0b63a5;
 
+// Indico logo colors
+$dark-blue-logo: #005272;
```

Comment:
`#00A4E4` was already defined as `$indico-blue`. Could you group the light and dark colors together outside of blue? Maybe move the group to the bottom/top of the file.